src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -197,7 +195,7 @@ def generate_FAD_2_default_config(self, filename):
         'conditionalRegistrationData': conditionals,
         'grantRequestG2': grant_g2,
         'grantRequestG4': grant_g4,
-        'palRecords': pal_records,
+        'palRecords': [pal_records],
```

Comment:
This is already a list so it doesn't need to be made a list again. We don't want a list of lists here. Please revert this change.

Reply:
reverted.